docs/conf.py

```diff
@@ -50,12 +51,16 @@
 copyright = f"2015–{datetime.utcnow().year}, {author}"  # noqa: A001, DTZ003
 language = "en"
 
+if release.startswith("0"):
+    warnings.warn(f"Incorrect version in documentation build ({release = })")
```

Comment:
We had been getting versions like `0.1.0dev50` in Read the Docs builds lately, so this is to make sure that we notice right away when this happens.